plugins/woocommerce/client/legacy/js/frontend/add-to-cart.js

```diff
@@ -169,6 +170,18 @@ jQuery( function( $ ) {
 		});
 	};
 
+	/**
+	 * Handle when pressing the Space key on the remove item link.
+	 * This is necessary because the link got the role="button" attribute
+	 * and needs to act like a button.
+	 */
+	AddToCartHandler.prototype.onKeydownRemoveFromCart = function( e ) {
+		if ( e.key === ' ' ) {
+			e.preventDefault();
+			$( this ).trigger( 'click' );
+		}
+	};
```

Comment:
For increased readability, I suggest using `event` instead of `e`. It's something we also do in other functions.

```suggestion
	AddToCartHandler.prototype.onKeydownRemoveFromCart = function( event ) {
		if ( event.key === ' ' ) {
			event.preventDefault();
			$( this ).trigger( 'click' );
		}
	};
```